integration-tests/src/test/java/oracle/weblogic/kubernetes/ItPortForward.java

```diff
@@ -0,0 +1,443 @@
+   * Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
+   */
+  @Order(1)
+  @Test
+  @DisplayName("Forward a local port to default channel port and verify WLS admin console is accessible")
+  @DisabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  void testPortForwardDefaultAdminChannel() {
+    final int adminDefaultChannelPort = 7001;
+
+    // Verify that using a local port admin forwarded to admin's default channel (7001),
+    // admin console is accessible via http://localhost:localPort/console/login/LoginForm.jsp
+    String portForwardFileName = portForwardFileNameProfix + "-1.out";
+    startPortForwardProcess(adminDefaultPortDomainNamespace,
+        adminDefaultPortDomainUid, adminDefaultChannelPort, portForwardFileName);
+    verifyAdminConsoleAccessible(adminDefaultPortDomainNamespace, portForwardFileName, false);
+  }
+
+  /**
+   * The test that `kubectl port-foward` is able to forward a local port to default secure channel port
+   * (7002 in this test).
+   * Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
+   */
+  @Order(2)
+  @Test
+  @DisplayName("Forward a local port to default secure channel port and verify WLS admin console is accessible")
+  @DisabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  void testPortForwardDefaultAdminSecureChannel() {
+    final int adminDefaultChannelPort = 7002;
+
+    // Verify that using a local port admin forwarded to admin's default secure channel (7002),
+    // admin console is accessible via https://localhost:localPort/console/login/LoginForm.jsp
+    String portForwardFileName = portForwardFileNameProfix + "-2.out";
+    startPortForwardProcess(adminDefaultPortDomainNamespace,
+        adminDefaultPortDomainUid, adminDefaultChannelPort, portForwardFileName);
+    verifyAdminConsoleAccessible(adminDefaultPortDomainNamespace, portForwardFileName, true);
+
+    // Verify that using a local port admin forwarded to admin's default channel (7001),
+    // admin console is still accessible via http://localhost:localPort/console/login/LoginForm.jsp
+    portForwardFileName = portForwardFileNameProfix + "-1.out";
+    verifyAdminConsoleAccessible(adminDefaultPortDomainNamespace, portForwardFileName, false);
+  }
+
+  /**
+   * The test that `kubectl port-foward` is able to forward a local port to WLS administration port
+   * (9002 in this test).
+   * Verify that the WLS admin console can be accessed using http://localhost:localPort/console/login/LoginForm.jsp
+   */
+  @Order(3)
+  @Test
+  @DisplayName("Forward a local port to WLS administration port and verify WLS admin console is accessible")
+  @DisabledIfEnvironmentVariable(named = "OKD", matches = "true")
+  void testPortForwardAdministrationPort() {
+    final int adminDefaultChannelPort = 9002;
+
+    // Verify that using a local port admin forwarded to administration port (9002),
```

Comment:
Description is already in the method level javdoc.

Reply:
done